wp-shopping-cart.php

```diff
@@ -123,11 +123,9 @@ public function setup_table_names() {
 		global $wpdb;
 		$wpdb->wpsc_meta                = WPSC_TABLE_META;
 		$wpdb->wpsc_also_bought         = WPSC_TABLE_ALSO_BOUGHT;
-		$wpdb->wpsc_region_tax          = WPSC_TABLE_REGION_TAX;
 		$wpdb->wpsc_coupon_codes        = WPSC_TABLE_COUPON_CODES;
 		$wpdb->wpsc_cart_contents       = WPSC_TABLE_CART_CONTENTS;
 		$wpdb->wpsc_claimed_stock       = WPSC_TABLE_CLAIMED_STOCK;
-		$wpdb->wpsc_currency_list       = WPSC_TABLE_CURRENCY_LIST;
```

Comment:
It looks like the constants are defined (https://github.com/wp-e-commerce/WP-e-Commerce/pull/1865/files#diff-66e9d8bfe5d295da98d32c2d9c5a8d5cR251), but the WPDB properties are not?